### DISCUSSION TITLE
Can not make sections work

### DISCUSSION THREAD

**Discussion by idemeum** [squidfunk/mkdocs-material#2165] [Q&A]
Hello - spent a lot of time, and can not quite make navigation sections to work. Was hoping someone could help and point me in the right direction. What am I doing wrong?

<img width="459" alt="image" src="https://user-images.githubusercontent.com/74406836/103433555-9ff22400-4ba7-11eb-919a-b88441018e15.png">

**Comment by squidfunk** [comment]
Could you please provide a minimal reproducible case and explain exactly what you're having trouble with? As an example for sections, check out the [changelog](https://squidfunk.github.io/mkdocs-material/changelog/). which lists a "Guides" section.

**Reply by idemeum** [reply]
Thank you @squidfunk . I am building a new documentation site and using version 6.2.3. I am trying to enable navigation.sections feature along with navigation.tabs. The yaml config file I am using as per the above screenshot. I do not have any errors, everything works locally, and I have tabs displayed, but the navigations.sections does not work. I tried all the things and not quite sure what I am missing here. Thank you!
